packages/datepicker/package.json

```diff
@@ -40,16 +41,19 @@
     "react": ">=0.15.0 < 17.0.0"
   },
   "devDependencies": {
-    "@pluralsight/ps-design-system-build": "^1.8.5",
-    "@pluralsight/ps-design-system-storybook-addon-theme": "^1.2.1",
-    "@pluralsight/ps-design-system-textinput": "^0.5.9",
+    "@pluralsight/ps-design-system-build": "*",
+    "@pluralsight/ps-design-system-storybook-addon-center": "*",
+    "@pluralsight/ps-design-system-storybook-addon-theme": "*",
+    "@pluralsight/ps-design-system-textinput": "*",
     "@storybook/addon-actions": "^4.0.0",
     "@storybook/addon-storyshots": "^4.0.0",
     "@storybook/react": "^4.0.0",
     "babel-cli": "^6.24.1",
     "babel-preset-env": "^1.6.0",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
+    "enzyme": "^3.2.0",
```

Comment:
What would you think of us migrating our project to react-testing-lib?  I like it because: avoids ginormous enzyme api and encourages more blackbox testing and avoids having to do an adapter for every react version.

Reply:
i have no concerns if you want to try it. i haven't used it before - i will check it out